Changelog.md

```diff
@@ -2,6 +2,11 @@
 
 Enhancements:
 
+* Allow perfectionists who write custom matchers to improve their
+  `failure_message` by using methods from RSpec::Matchers::Pretty,
+  now renamed to RSpec::Matchers::English.  Before this, the module
```

Comment:
This is a bit verbose, should wrap constant names in backticks and you have an excess space after a period. How about:

```
* Expose `RSpec::Matchers::EnglishPhrases` to make it easier to write nice failure messages in custom matchers.
```